code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -18,6 +18,7 @@
 	force = 200
 	armour_penetration = 100
 	anchored = TRUE
+	flags = HANDSLOW
```

Comment:
doing this removes any flags it may already have, are you sure you want it to be JUST HANDSLOW?